dll/ime/msctfime/msctfime.rc

```diff
@@ -0,0 +1,25 @@
+#include <windef.h>
```

Comment:
Shouldn't we add the standard file headers also in the resource files (this one and the en-US.rc one)?
cc @binarymaster and others.

Reply:
Added.